oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -91,13 +104,13 @@ async fn test_server() {
         );
     }
     {
-        // Valid location but no lookup match.
-        let response = make_request(server_port, br#"{"lat":19,"lon":88}"#)
+        // Close to key_1.
```

Comment:
Perhaps also add a test for an exact location match.

Reply:
Done.